### PR TITLE
Rename QR code support option

### PR DESCRIPTION
**Pull request by data-man**
Proposal: include libqrcodegen sources to your library.

### CODE REVIEW

**Comment by dankamongmen** [comment]
I have it as "Disable" because the default option is ON, i.e. you only need to mess with this to disable the support. I think making it off by default might be sensible, though, especially if #1282  ever happens. In that case, this would indeed be "enable".

File: CMakeLists.txt [hunk 1]
```diff
@@ -26,7 +26,7 @@ cmake_dependent_option(
 option(USE_DOXYGEN "Build HTML cross reference with doxygen" OFF)
 option(USE_PANDOC "Build man pages and HTML reference with pandoc" ON)
 option(USE_POC "Build small, uninstalled proof-of-concept binaries" ON)
-option(USE_QRCODEGEN "Disable libqrcodegen QR code support" ON)
+option(USE_QRCODEGEN "Enable libqrcodegen QR code support" ON)
```
Comment (dankamongmen) [inline CMakeLists.txt:29]: i'm down with this, but change the ON to OFF in this case. the help ought be saying what the option can do to change from the defaults. it only makes sense for this to be "enable" if the default is OFF.

**Comment by dankamongmen** [comment]
Good deal; this has always annoyed people, and I don't think anyone really uses it.

**Comment by dankamongmen** [comment]
you gonna follow this up by making some Zig wrappers? =]

**Comment by data-man** [comment]
No, I have no experience in that.

**Comment by dankamongmen** [comment]
> No, I have no experience in that.

oh, my mistake, i thought your github page showed a bunch of Zig. well, thanks for the patch!

**Pull request merged**